app/src/routes/event/[id]/+page.server.ts

```diff
@@ -19,10 +20,18 @@ export const load: PageServerLoad = async ({ params: { id }, locals: { loadQuery
   const form = await superValidate(zod(registrationSchema));
   const initial = await loadQuery<Event>(query, { id });
 
+  const eventId = (await getEvent(id)).data?.event_id;
+  let eventParticipants: { full_name: string | null }[] = [];
```

Comment:
Tror man kan unngå let. For eksempel om participants kan være null.

Reply:
Hmm fikk ikke helt til å unngå det, for kan jo sjekke om den finnes og så returnere `fail` hvis ikke, men siden det er i load funksjonen så ødelegger det for resten. Eventet bør jo kunne lastes inn selvom deltagerlisten feiler, så må i såfall skrive om litt der

Reply:
Okey skjønner utfordringen 🤔 

Man kan gjøre det på denne måten ved å gå gjennom 'event' tabellen istedenfor:

```
// queries.ts
export const getEventParticipantNames = async ({
  document_id,
}: Pick<Tables<"event">, "document_id">) => {
  const result = await supabase
    .from("event")
    .select("event_participant(full_name)")
    .eq("document_id", document_id)
    .maybeSingle();

  return result.data?.event_participant.map(({ full_name }) => full_name);
};

```
// +page.server.ts
const participantNames = await getEventParticipantNames({
  document_id: id,
});

```

Reply:
Eventuelt, men ikke nødvendigs penere kunne man latt event_id vært optional:
```
export const getEventParticipantNames = async ({
  event_id,
}: Partial<Pick<Tables<"event_participant">, "event_id">>) => {
  if (!event_id) return [];
  ..
```

Reply:
Takk for tips 😄 Skal fikse!